controllers/main.js

```diff
@@ -21,12 +21,14 @@ module.exports.renderPlaySettings = (req, res) => {
 module.exports.renderPlay = async (req, res) => {
   const image = await getPixelatedImage(req.gameData.background_image);
   const playSessionData = await fetchPlaySessionData(req.user.username);
+  const pageNum = playSessionData.sessionData.length + 1;
   res.render("main/play.ejs", {
     image,
     gameName: req.gameData.name,
     imgLink: req.gameData.background_image,
     extraStyles: cardStyle,
     fetchedSession: playSessionData,
+    pageNum: pageNum,
```

Comment:
Also here, for the sake of performance on client side and simplicity of the code, I think it would be better to pass needed data only. From my understanding, only session id and array length of `playSessionData` are used in `play.ejs` template. I would pass session length number and session id seperately.